code/modules/mob/living/simple_animal/friendly/mouse.dm

```diff
@@ -88,22 +88,23 @@
 		evolve()
 		qdel(AM)
 
-/mob/living/simple_animal/mouse/handle_automated_action()
+/// How miceys go nom nom nom on things. force_chew is used to bypass the probability check (for unit tests).
+/mob/living/simple_animal/mouse/handle_automated_action(force_chew = FALSE)
```

Comment:
We shouldn't be adding new named parameters like this to the base of `handle_automated_action`. Make `force_chew` a var on the mouse instead and instantiate a custom subtype that only exists in tests.

Reply:
Better yet, make a proc called `should_chew()` which returns `prob(15)` on base and `TRUE` on our subtype.

Reply:
i just offloaded all of the probability logic to the new proc